test/common/lcov.py

```diff
@@ -392,6 +392,17 @@ def get_review_comments(diff_info_file):
     start_line = None
     cur_line = None
 
+    def is_interesting_line(text):
+        # Don't complain when being told to shut up
+        if "// not-covered: " in text:
+            return False
+        # Don't complain about lines that contain only punctuation, or
+        # nothing but "else".  We don't seem to get reliable
+        # information for them.
+        if not re.search('[a-zA-Z0-9]', text.replace("else", "")):
```

Comment:
I'm a bit concerned about accidentally ignoring lines that have "else" as a sub-word, like "panelselector" or so, but I think it's okay to ignore that :grin:

Reply:
No, this will remove "else" from a line,. and will ignore it if there are no letters or digits left.  So, it wont ignore "panelselector" since "panlector" still has some letters. But it will ignore lines like "}" and "} else {".

Reply:
Ah, right!